core/src/main/java/org/commonjava/indy/core/inject/IspnNotFoundCache.java

```diff
@@ -85,47 +77,24 @@ private void start()
         maxResultSetSize = config.getNfcMaxResultSetSize();
     }
 
-    private int getTimeoutInSeconds( ConcreteResource resource )
-    {
-        int timeoutInSeconds = config.getNotFoundCacheTimeoutSeconds();
-        Location loc = resource.getLocation();
-        Integer to = loc.getAttribute( RepositoryLocation.ATTR_NFC_TIMEOUT_SECONDS, Integer.class );
-        if ( to != null && to > 0 )
-        {
-            timeoutInSeconds = to;
-        }
-        return timeoutInSeconds;
-    }
-
     @Override
     public void addMissing( final ConcreteResource resource )
     {
-        final int timeoutInSeconds = getTimeoutInSeconds( resource );
```

Comment:
Home-made expiration code is not needed anymore. Will only use ISPN expiration config.